tests/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+set(UNITTESTS_SRC
+        unittests/unittests.cpp
+        unittests/mallocmacro.cpp
+        common/simplethread.cpp
+        common/systemtime.cpp
+)
+add_executable(unittests ${UNITTESTS_SRC})
+target_link_libraries(unittests PRIVATE concurrentqueue)
+add_test(NAME unittests COMMAND unittests)
```

Comment:
`unittests` command has `--disable-prompt` flag, I think it will be good to add such flag here to remove `Press enter to continue` at the end of tests.